pkg/stacks/unpack.go

```diff
@@ -498,16 +498,15 @@ func (sp *StackPackage) applyRules() error {
 			}
 
 			gk := schema.ParseGroupKind(crd)
-			if gk.Group == "" || gk.Kind == "" {
```

Comment:
help me understand this change, why do we no longer check for `gk.Group == ""` here?

Reply:
We are no longer preventing the use of the corev1 apigroup during stackinstall's unpack. We defer those decisions to the stack controller.

Unpack could he aware of the restriction mode but we would have to snake that setting through the stackinstalls controller and the unpack command. Would we repeat (pre-peat?) all of the restriction checking at unpack time?

Reply:
ha oh right, duh :)

so you were right, we did have a tiny bit of core api group checking code already in the code base! i think this is the right move to consolidate the enforcement in the stack controller, before generating any rbac policies on behalf of the stack.  this is good.